src/components/autosuggest-material/autosuggest-material.tsx

```diff
@@ -0,0 +1,82 @@
+import { UseComboboxPropGetters } from "downshift";
+import React from "react";
+import { generateItemId } from "../autosuggest-text/autosuggest-text";
+import {
+  Suggestion,
+  SuggestionWork
+} from "../autosuggest-text/autosuggest-text-item";
+
+export interface AutosuggestMaterialProps {
+  materialData: SuggestionWork[] | [];
+  getItemProps: UseComboboxPropGetters<Suggestion>["getItemProps"];
+  highlightedIndex: number;
+  textDataLength: number;
+}
+
+const AutosuggestMaterial: React.FC<AutosuggestMaterialProps> = ({
+  materialData,
+  getItemProps,
+  highlightedIndex,
+  textDataLength
+}) => {
+  return (
+    <>
+      <li>
+        <hr className="autosuggest__divider" />
+      </li>
+      <li>
+        <ul className="autosuggest__item--materials">
+          {/* eslint-disable react/jsx-props-no-spreading */}
+          {/* The downshift combobox works this way by design */}
+          {materialData.map((item, index) => {
+            let numberOfAuthors = 0;
+            return (
+              <li
+                className={`autosuggest__item--materials__item ${
+                  highlightedIndex === index + textDataLength
+                    ? "autosuggest__item--materials__item--highlight"
+                    : ""
+                }`}
+                key={generateItemId(item)}
+                {...getItemProps({ item, index })}
+              >
+                {/* eslint-enable react/jsx-props-no-spreading */}
+                <div className="autosuggest__item--materials__item__content">
+                  <div className="autosuggest__item--materials__item__content--cover">
+                    <div className="material-container">
+                      <a
+                        className="material material--xsmall bg-identity-tint-120 material__animate"
+                        href="google.com"
+                      >
+                        <img
+                          src="https://imgcdn.saxo.com/_9781616550417/0x500"
```

Comment:
Why not have generic cover placeholders in design-system. Linking to extern CDN that may change over time is no-go.

Reply:
Yes that's true, but we just need something that works for the nearest couple days. Kasper and Kasper have been working on the cover service calls, so this will be updated as soon as it is merged in to their new changes. I am writing a comment in the code for this one.